core/commands/root.go

```diff
@@ -38,6 +38,9 @@ BASIC COMMANDS
   ls <ref>      List links from an object
   refs <ref>    List hashes of links from an object
 
+BTFS COMMANDS
+  storage       Store a file in nodes
```

Comment:
`storage upload` is the command to store a file

`storage` contains other commands that will be used for other purposes